include/WPA/Andersen.h

```diff
@@ -65,6 +66,10 @@ class AndersenBase:  public WPAConstraintSolver, public BVDataPTAImpl
         if (consCG != nullptr)
             delete consCG;
         consCG = nullptr;
+
+        auto * chg = getCHGraph();
+        delete chg;
+        chg = nullptr;
```

Comment:
Would be good to move this destructor to cpp file. `#include "SVF-FE/CommonCHG.h"` can also be removed.

Reply:
Done.